literals/literals.go

```diff
@@ -0,0 +1,240 @@
+package literals
```

Comment:
please move these to internal packages, like `internal/literals`; otherwise people might assume that these are stable APIs they can rely on from external modules.